src/InputWithLabel.js

```diff
@@ -0,0 +1,26 @@
+import React from'react';
```

Comment:
can import useRef and useEffect from react since they are named exports:
import React, { useRef, useEffect } from 'react';